lib/urn.rb

```diff
@@ -7,19 +13,20 @@ class URN
   private :urn
 
   def initialize(urn)
+    fail(InvalidURNError, "bad URN(is not URN?): #{urn}") if urn !~ REGEX
```

Comment:
While `fail` is a method call, would it be more idiomatic to skip the parentheses ala `raise`?

```ruby
fail InvalidURNError, "bad URN(is not URN?): #{urn}" if urn !~ REGEX
```